turbo/engineapi/engine_block_downloader/block_downloader.go

```diff
@@ -238,66 +237,3 @@ func saveHeader(db kv.RwTx, header *types.Header, hash libcommon.Hash) error {
 	}
 	return nil
 }
-
-func (e *EngineBlockDownloader) insertHeadersAndBodies(ctx context.Context, tx kv.Tx, fromBlock uint64, fromHash libcommon.Hash, toBlock uint64) error {
```

Comment:
This function was removed, as its functionality was moved to the function `downloadAndLoadBodiesSyncronously` which now inserts blocks via the same API as Caplin, without the need to accumulate them in the tempDb (and potentially cause `MDBX_MAP_FULL`)